src/composeappmanager.cc

```diff
@@ -221,6 +225,7 @@ ComposeAppManager::AppsContainer ComposeAppManager::getAppsToUpdate(const Uptane
         !boost::filesystem::exists(cfg_.apps_root / app_name / Docker::ComposeAppEngine::ComposeFile)) {
       // an App that is supposed to be installed has been removed somehow, let's install it again
       apps_to_update.insert(app_pair);
+      apps_and_reasons.push_back(app_pair.first + " (removed)");
```

Comment:
Maybe something like: "app: missing installation, to be re-installed".

Reply:
Updated.